lib/exercises.rb

```diff
@@ -2,19 +2,46 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n * m) where n is the number of characters in the longest word and m is the size of the input array
+# It might actually simplify to just O(n) where n is the total number of input charcters, because you're only sorting each word once
+# Space Complexity: O(n) where n is the size of the input array
 
 def grouped_anagrams(strings)
```

Comment:
👍 You are right on time complexity except that it would be better said that time complexity is O(n) if the words are guaranteed to be less than a certain length.